frontend/src/tests/mocks/dataMocks.js

```diff
@@ -2437,3 +2475,74 @@ export const VARIANCE_DOCUMENT_CATEGORY_OPTIONS_HASH = {
   REC: "Recommendation Document",
   DEC: "Decision Document",
 };
+
+export const INCIDENT_STATUS_OPTIONS = [
+  { description: "Preliminary", display_order: 10, mine_incident_status_code: "PRE" },
+  {
+    description: "Final",
+    display_order: 20,
+    mine_incident_status_code: "FIN",
+  },
+];
+
+export const INCIDENT_STATUS_OPTIONS_HASH = {
+  PRE: "Preliminary",
+  FIN: "Final",
+};
+
+export const INCIDENT_DETERMINATION = [
+  {
+    active_ind: true,
+    description: "Pending determination",
+    display_order: 10,
+    mine_incident_determination_type_code: "PEN",
+  },
+  {
+    active_ind: true,
+    description: "This was a dangerous occurrence",
+    display_order: 20,
+    mine_incident_determination_type_code: "DO",
+  },
+  {
+    active_ind: true,
+    description: "This was not a dangerous occurrence",
+    display_order: 30,
+    mine_incident_determination_type_code: "NDO",
+  },
```

Comment:
`active_ind` and I believe `display_order` are not meant to be exposed by the API. I'm assuming you've added them here because they're being returned?

If they are, I'll update the follow-up backend ticket that I made to include properly filtering out inactive records & withholding these two fields

Reply:
They are indeed being returned, I can delete them from the mocks as they are not being used on the UI.

Reply:
Okay. I'll update the follow-up ticket to include this new info